Header header
std_msgs/String Part
std_msgs/Float32 Angles1
std_msgs/Float32 Angles2

